ui/apps/platform/src/Containers/Dashboard/ScopeBar.cy.jsx

```diff
@@ -0,0 +1,141 @@
+import React from 'react';
+
+import ComponentTestProviders from 'test-utils/ComponentProviders';
+import { graphqlUrl } from 'test-utils/apiEndpoints';
+
+import ScopeBar from './ScopeBar';
+
+const mockData = {
+    clusters: [
+        {
+            id: `cluster-0`,
+            name: 'production',
+            namespaces: [
+                { metadata: { id: 'ns-0', name: 'backend' } },
+                { metadata: { id: 'ns-1', name: 'default' } },
+                { metadata: { id: 'ns-2', name: 'frontend' } },
+                { metadata: { id: 'ns-3', name: 'kube-system' } },
+                { metadata: { id: 'ns-4', name: 'medical' } },
+                { metadata: { id: 'ns-5', name: 'payments' } },
+            ],
+        },
+        {
+            id: `cluster-1`,
+            name: 'security',
+            namespaces: [
+                { metadata: { id: 'ns-0', name: 'default' } },
+                { metadata: { id: 'ns-1', name: 'kube-system' } },
+                { metadata: { id: 'ns-2', name: 'stackrox' } },
+            ],
+        },
+    ],
+};
+
+const setup = () => {
```

Comment:
Although `function` versus arrow function in itself does not affect readability for me, it looks like `function setup` has the majority in this contribution. Sometimes I use old school Find in Files for other examples.